recipes/sompy/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "sompy" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "846626a39e27cca171a81f4388cb30a32cdcd3bedf170c91227cf5a958aaabec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
```

Comment:
It doesn't look like `numpy` is a `build` requirement if using `--single-version-externally-managed`